test/fixtures/developers.yml

```diff
@@ -45,6 +45,13 @@ with_full_time_employment:
   name: Full-time Employee
   hero: A full-time employee
   bio: I am a developer available for full-time employment.
+
+invisible:
+  user: with_invisible_profile
+  name: Invisible Developer
+  hero: I'd rather stay invisible for now
+  bio: Soon to be visible
+  search_status: "invisible"
   search_status: open
```

Comment:
Unless I'm reading this diff wrong, it looks like the second `search_status` will override the first one.

I think `open` should be moved above (where it was) and `invisible` should be here. Also, you can drop the quotes.